tools/wptrunner/wptrunner/wpttest.py

```diff
@@ -589,6 +593,13 @@ def fuzzy_override(self):
         return values
 
 
+class PrintReftestTest(ReftestTest):
+    test_type = "print-reftest"
+
+    def get_viewport_size(self, override):
+        assert override is None
+        return (5*2.54, 3*2.54)
```

Comment:
So it seems like the webdriver API is in cm, whilst we spec'd this in inches. I presume most paginated media uses inches?

Any thoughts on converting here vs having `viewport_size` in inches and converting close to where we call webdriver?

Reply:
I don't think "most paginated media uses inches"; all the A* paper sizes are specified in metric. AFAICT the use of inches by default is basically US-bias in the underlying API. I don't really want to use inches in the external facing parts.

Reply:
I don't follow here - which underlying API are you referring to?